mongo/risk_assessment_service.go

```diff
@@ -0,0 +1,52 @@
+package mongo
+
+import (
+	"time"
+
+	"gopkg.in/mgo.v2/bson"
+
+	"github.com/intervention-engine/fhir/models"
+	"github.com/intervention-engine/ie/app"
+)
+
+// RiskAssessmentService List and transform FHIR RiskAssessments
+type RiskAssessmentService struct {
+	Service
+}
+
+// RiskAssessments List risk assessments for a patient
+func (ras *RiskAssessmentService) RiskAssessments(patientID string, serviceID string, start time.Time, end time.Time) ([]*app.RiskAssessment, error) {
+
```

Comment:
I believe this should have a `defer ras.S.Close()` in order to close the session this RiskAssessmentService has when this function is done.